src/lib.rs

```diff
@@ -10,7 +10,6 @@
 //! * Serde support with the `serde` feature.
 
 #![forbid(unsafe_code)]
-#![warn(clippy::all)]
```

Comment:
I guess this was redundant with the default config, is that right?

Reply:
I'm not sure why exactly I removed that. I thought it was due to some warning of RA, but it doesn't produce any warnings anymore now.

Maybe I just removed it because it caused a lot of annoying errors while working on it.

Anyways ... since I'm in doubt, I'll just reintroduce it. It certainly won't hurt anyone. Thanks for spotting that!

Reply:
Strange indeed. Not sure what my train of thought was there :sweat_smile:

Reply:
Maybe you put it there to force people with less than default strict configs to use the defaults? 😅